packages/@aws-cdk/aws-cloudwatch/README.md

```diff
@@ -291,6 +291,75 @@ new cloudwatch.CompositeAlarm(this, 'MyAwesomeCompositeAlarm', {
 });
 ```
 
+## Contributor Insight Rules
```

Comment:
Does this contribution build off of and/or use code from any of the original PRs? If (likely) so, then they should be properly given credit in the PR description: https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors